packages/create-instantsearch-app/README.md

```diff
@@ -50,6 +50,10 @@ cd my-app
 yarn start
 ```
 
+> **Important**
+> Selecting 'Y' when being prompted to ‘Enable user events’ activates the [`insights`](https://www.algolia.com/doc/api-reference/widgets/instantsearch/js/#widget-param-insights) option for compatible templates, allowing Algolia to process your user Events. Events can unlock powerful features, enhancing your application's effectiveness and we encourage you to consider enabling this valuable functionality. Please review our [API reference](https://www.algolia.com/doc/api-reference/widgets/instantsearch/js/#widget-param-insights) for more details about Events collection and settings.
```

Comment:
should it mention `--yes` or non-interactive too?

Reply:
I think this formulation works as is : CISA doesn't handle `--yes` and `--no-interactive` skips the question (`enableInsights` is undefined).